models/comment.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose')
+
+const commentSchema = mongoose.Schema({
+	_id: mongoose.Schema.Types.ObjectId,
+    email: {
+    	type: String,
+    	required: true,
+    },
+    commentText: {
+    	type: String,
+    	required: true,
+    },
+    blogId: {
```

Comment:
As mention in comment of email. BlogId also needs ref.

Reply:
As of now, blog collection, does not exist, so I will create one collection with `ObjectId` field only. Will add TODO comment in code to add more fields.

Reply:
Okay that works